marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx

```diff
@@ -513,6 +513,17 @@ describe('Verify numeric facet can be applied', () => {
         browsePage.getFacetApplyButton().click();
         browsePage.getRangeFacet(3024).should('exist');
         browsePage.getClearAllButton().should('exist');
+        browsePage.getClearAllButton().click();
+        //Verify clearing date range facet clears corresponding selected facet
+        browsePage.selectEntity('Customer');
```

Comment:
@timur-isangulov once we use clearAllButton the page will come back to it's original state right ? Did you have any reason to repeat all the statements from 518-520 ?

Reply:
Removed duplicate lines. Thanks.